frontend/pnpm-lock.yaml

```diff
@@ -4,11 +4,11 @@ settings:
   autoInstallPeers: true
   excludeLinksFromLockfile: false
 
-pnpmfileChecksum: sha256-QrioRE0rVa7k37ba9jgFOwNQflzbUTpdxTezotowvIE=
+pnpmfileChecksum: eqzj7c6t3bs5v7xcjer2wfwkp4
```

Comment:
this change might be b/c of a different pnpm version. you could do `make check-prereqs`

Reply:
yep, i'm already on it.  updating to pnpm 10 and node 22 (as the release workflow specifies) fixes the issue.